monkey/tests/unit_tests/common/utils/test_common_file_utils.py

```diff
@@ -34,6 +35,10 @@ def test_get_file_sha256_hash(stable_file, stable_file_sha256_hash):
     assert get_file_sha256_hash(stable_file) == stable_file_sha256_hash
 
 
+def test_get_binary_sha256_hash(stable_binary, stable_binary_sha256_hash):
+    assert get_binary_sha256_hash(stable_binary) == stable_binary_sha256_hash
```

Comment:
We can simplify this. Rather than store something on disk and then open it, wrap it in a fixture, etc, we can just pass in a BytesIO:

```suggestion
def test_get_binary_sha256_hash():
    expected_hash = "XXXX"
    assert get_binary_sha256_hash(BytesIO(b'Hello World!')) == expected_hash
```